### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nunoguedelha**
Update references to the `dump_data`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,19 +118,21 @@ export YARP_DATA_DIRS=$YARP_DATA_DIRS:$WalkingControllers_INSTALL_DIR/share/yarp
...
-`WalkingJoypadModule` in the same machine where the physical device is
-connected. The only think that you have to do is running the following command from
-the terminal
+The Joypad application, called `WalkingJoypadModule`, allows you to send all the rpc commands using the buttons. The application processes the button press events associating them to the pre-defined rpc commands which are then sent through Yarp to the Walking Coordinator module. The joypad keys mapping is as follows:
+ * `A` for preparing the robot
+ * `B` for start walking
+ * `Y` for pause walking
+ * `X` for stop walking
+
+Suppose that you want to run the Joypad application, called `WalkingJoypadModule` in the same machine where the physical device is connected. The only think that you have to do is running the following command from the terminal:
```
Comment (S-Dafarra) [inline README.md:127]:
```suggestion
Suppose that you want to run the Joypad application, called `WalkingJoypadModule` in the same machine where the physical device is connected. The only thing that you have to do is running the following command from the terminal:
```

**Comment by nunoguedelha** [comment]
sorry @S-Dafarra , don't bother approving again, another link was broken. Fixing it now...

**Review by S-Dafarra — APPROVED** [review]
Thanks @nunoguedelha, LGTM! @GiulioRomualdi you might also want to take a look at this

**Review by GiulioRomualdi — APPROVED** [review]

**Comment by GiulioRomualdi** [comment]
Thank you @nunoguedelha

**Pull request merged**